src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.js

```diff
@@ -132,7 +132,7 @@ class DropdownItem extends React.Component {
         )}
         title={title}
       >
-        {Icon && <Icon left outline />}
+        {Icon && <Icon className="DropdownItem__button-icon" left outline />}
```

Comment:
Change spacing in Icon with `left` or `right` prop

Reply:
Done